cocos2d/Platforms/Android/CCGLView.h

```diff
@@ -41,9 +41,8 @@ enum CCAndroidScreenMode {
     CCScreenScaledAspectFitEmulationMode
 };
 
-BRIDGE_CLASS("com.apportable.GLView")
-@interface CCGLView : GLView <CCDirectorView>
-
+BRIDGE_CLASS("org.cocos2d.CCGLView")
```

Comment:
I'm going to CC @osinoleg on this. I haven't looked into any of his android changes here. I know he's working on it.